### DISCUSSION TITLE
Longterm enhancement proposal

### DISCUSSION THREAD

**Discussion by Naeemkh** [NSAPH/handbook#4] [Ideas]
1. Thanks, @atrisovic, for putting this handbook together. We need to keep a couple of points in mind for future development.  
2. With growing the team and hammering the processing steps, we may need to change the steps in the projects. Those people who are in the middle of the project need to follow the old style; those who are about to start a project need to use the new confirmed style. As a result, we need to think about a version of the handbook and make sure that we can render different versions.  
3. Let's keep the master branch for the most reviewed documents and use develop branch for ongoing activities. By expanding the team, rapid changes will have consequences, so it will require good revision. 
4. Also, we need to add steps to render the book locally for developers.

**Comment by atrisovic** [comment]
All good points! 
We needed to start from somewhere (but I'm really open to suggestions!) :)

**Reply by Naeemkh** [reply]
@atrisovic, sure it is an ongoing process.
